enhancements/cloud-integration/tokenized-auth-enablement-operators-on-cloud.md

```diff
@@ -353,8 +358,34 @@ On AWS STS, the `STSIAMRoleARN` would be something like:
 `arn:aws:iam::269733383066:role/newstscluster-openshift-logging-role-name`<br>
 This provides the role information linked by policy to access various cloud resources.
 
+### Changes for HyperShift
+
+#### CCO Stories
+
+The kubeconfig of the hosted cluster is mounted at a specied path on the CCO 
+pod. CCO can consume that kubeconfig when it starts the go client.
+
+The Pod Identity Webhook is not shipped alongside CCO in order to reduce the 
+footprint of the CCO install.
+
+#### HyperShift Stories
+
+CCO without the pod identity webhook is enabled on the HyperShift management 
+plane.
+
+When a new hosted cluster is created, a new instance of CCO is deployed with 
+the kubeconfig of that cluster mounted to the pod.
+
 ### Risks and Mitigations
 
+Enabling CCO on HyperShift incurs additional infrastructure cost. The exact 
```

Comment:
It'd be good to have concrete data for:
- Compute resources impact
- RBAC reqs management and guest cluster side
- Networking requirements

Reply:
> Compute resources impact

We've done some aggressive pruning and the final numbers are low. The CCO container and the rbac-proxy together are something like 80MiB RAM and 20m vCPU. I'm going to work on the RAM a bit more after this, I think we should be able to get that down to 20MiB or so.